plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/client/TestElasticsearchClient.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.elasticsearch.client;
+
+import io.airlift.units.Duration;
+import io.trino.plugin.elasticsearch.ElasticsearchConfig;
+import org.testng.annotations.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
+import static java.util.concurrent.TimeUnit.MINUTES;
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static org.testng.Assert.assertEquals;
+
+public class TestElasticsearchClient
+{
+    @Test
+    public void test() throws IOException
```

Comment:
This test is unnecessary. It doesn't validate anything about the state of the client or that it works -- just that it can be constructed.

Let's remove it.

Reply:
@martint  Thanks for your comments! delete the test.